tests/test_encode.py

```diff
@@ -77,7 +82,12 @@ def assert_rgb_rotate(self, input_):
         stream.average_rate, 24
     )  # Only because we constructed is precisely.
     self.assertEqual(stream.rate, Fraction(24, 1))
-    self.assertEqual(stream.time_base * stream.duration, 2)
+    if stream.duration is not None:
```

Comment:
I'm not keen of these changes, I don't like tests which depend on the output. How about adding an explicit `is_dash` argument to assert_rgb_rotate?

Reply:
Changed in https://github.com/PyAV-Org/PyAV/pull/718/commits/d1a9c422924d613c2bc9996828b12da46bf6e273. I noted a build failure on FFmpeg 4.1 and have added a fix for that too.

Reply:
Looks good!